src/Context/FeatureContext.php

```diff
@@ -741,6 +823,11 @@ private function set_cache_dir() {
 	 * @param bool $add_database Optional. Whether to add dbname to the $sql_cmd. Default false.
 	 */
 	private static function run_sql( $sql_cmd, $assoc_args = [], $add_database = false ) {
+		/* Can't run SQL against SQLite yet */
+		if ( 'sqlite' === self::$db_type ) {
+			return;
```

Comment:
Should we throw some form of exception if the method gets called with `self::$db_type`, instead of bailing early?

If the method gets called and isn't supported, I'd think we'd want to noisily fail.

Reply:
I did put this in here mostly as a fallback and also maybe as a reminder/suggestion that perhaps it should be possible to run some direct sql code against a SQLite database in the future.

Also When I was originally testing a lot of the sql code was for stuff that just isn't needed with SQLite, so it was actually fine to just skip and things would complete as you'd hope. I ended up trying to put all of those calls in a `if( mysql === 'self::$db_type )` check for clarity so this was probably less necessary but still left in.

I can remove if that would be better.

Reply:
@mrsdizzie Yeah, I think it'd be better to remove. I'd rather it break noisy if someone tries to call it instead of silently fail.

Reply:
Removed in 92f119e5c4c7cd427f0732de3ae7aaf22725316d